lib/mumuki/domain/extensions/hash.rb

```diff
@@ -1,5 +1,14 @@
 class Hash
-  def markdownify!(*keys)
-    keys.each { |it| self[it] = Mumukit::ContentType::Markdown.to_html(self[it]) }
+  def markdownify!(*keys, **options)
+    warn "Don't use markdownify. Use markdownified! instead"
+    markdownified! *keys, **options
+  end
+
+  def markdownified!(*keys, **options)
+    keys.each { |it| self[it] = self[it].markdownified }
```

Comment:
is it ok for options to be ignored here? :monocle_face:

Reply:
nope!!! My bad

Reply:
Solved!